YAXLib/Attributes/YAXSerializableTypeAttribute.cs

```diff
@@ -81,6 +82,12 @@ public YAXSerializationOptions Options
             }
         }
 
+        /// <summary>
+        ///     If set to <see langword="true"/>, private fields from <see cref="Type.BaseType"/>s
+        ///     can be searched for fields to serialize. Default is <see langword="false"/>.
+        /// </summary>
+        public bool IncludePrivateMembersFromBaseTypes { get; set; }
```

Comment:
Maybe this should be a member of YAXLib/Enums/YAXSerializationOptions.cs - it's already a flags enum.

(member of YAXSerializationFields would be better, but that's is a really large refactor)

Reply:
Good point, yep

Reply:
If we use `YAXSerializationOptions`, the setting applies for the whole object graph to serialize. That's what lead to failing unit tests when trying to always include private members from base types.
Agree that `YAXSerializationFields` better were be a flags enum. As it isn't `YAXSerializableTypeAttribute.IncludePrivateMembersFromBaseTypes` became the workaround. What's the way to go:
Bite the bullet and refactor `YAXSerializationFields`?
Leave the workaround?
Move to `YAXSerializationOptions`?

Reply:
I think we could leave if like this. I don't think there are more reasons to refactor all the stuff. So a bit overkill